src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -33,7 +33,7 @@
             + PREFIX_FEED_TIME + "1400 ";
 
     public static final String MESSAGE_SUCCESS = "New animal added: %1$s";
-    public static final String MESSAGE_DUPLICATE_ANIMAL = "This animal already exists in the address book";
+    public static final String MESSAGE_DUPLICATE_ANIMAL = "An animal with this ID already exists in the address book";
```

Comment:
Should be zookeep book (or can just say "An animal with this ID already exists")

Reply:
Error messages and json updated